02-lists/02-lists.md

```diff
@@ -0,0 +1,20 @@
+# lists 
+
+Welcome to lists exercise!
+
+## Verifying your work
+
+Run `python3 test_lists.py` to confirm that you have succesfully created the required functions.
+
+## sort 
```

Comment:
`sort` shadows the Python built-in function [sort](https://www.w3schools.com/python/ref_list_sort.asp). Is this intentional?

Reply:
This needs to refactored. Excellent catch!